### ISSUE TITLE
Mobile Wallet

### ISSUE THREAD

**Issue by borislavkharlamov** [PandoraPay/PandoraPay-wallet#39]
Please have a look at the [repository](https://github.com/borislavkharlamov/pandora-pay-flutter) I created. This latest repository enables the web wallet to function as a native application on both Android and iOS platforms. Additionally, it supports socks5 proxy. You ought to fork it and transfer it to your GitHub account. Kindly assess my work and approve the pull requests.

**Comment by ibudisteanu** [comment]
I don't think it is necessary to have a mobile app. The front-end runs responsive on any smart phone's browser (chrome, safari, firefox).

**Comment by ibudisteanu** [comment]
I also think the electron app is safer running on desktops like Linux distros than on Android/iOS

**Comment by ibudisteanu** [comment]
Wouldn't a mobile app built in Ionic or React native be easier to maintain and develop ?

**Comment by borislavkharlamov** [comment]
My expertise lies solely in Flutter. However, Ionic, React Native, and Xamarin are alternative frameworks that can be utilized to develop mobile wallets with similar or additional functionalities. I am uncertain whether the app can obtain access to the ProxyController in React Native or Ionic.
